torchtune/utils/quantization.py

```diff
@@ -22,7 +22,19 @@
 
 
 if TORCH_VERSION_AFTER_2_3:
-    from torchao.quantization.quant_api import Int8DynActInt4WeightQuantizer
+    from torchao.quantization.quant_api import (
+        int8_dynamic_activation_int4_weight,
+        quantize_,
+    )
+
+    class Int8DynActInt4WeightQuantizer:
```

Comment:
I know it's unrelated to this PR, but can I request a doc string here which explains this a bit more. Pointing back to ao docs is fine too